sql/new-install/MinimalData.sql

```diff
@@ -5,7 +5,7 @@ INSERT INTO users (email, first_name, last_name, institution, created, password,
 	('admin@uiowa.edu', 'Admin', 'User', 'The University of Iowa', SYSDATE(), 'b109f3bbbc244eb82441917ed06d618b9008dd09b3befd1b5e07394c706a8bb980b1d7785e5976ec049b46df5f1326af5a2ea6d103fd07c95385ffab0cacbc86', 107374182400),
 	('public', 'Public', 'User', 'None', SYSDATE(), 'd32997e9747b65a3ecf65b82533a4c843c4e16dd30cf371e8c81ab60a341de00051da422d41ff29c55695f233a1e06fac8b79aeb0a4d91ae5d3d18c8e09b8c73', 52428800);
 
-INSERT INTO user_roles VALUES
+--INSERT INTO user_roles VALUES
```

Comment:
I am not sure about this edit.  It looks like the file will not be well-formed sql any more.  Now, you can reasonably wish the sample data not to be uiowa-based...  But maybe this is not too important at the moment.  This -- seems to be spurious, though.

Reply:
I agree. We've customized this template with hard-coded values specific to our site. Perhaps we can make a note for future institutions/devs that this file should be considered as a template, similar to the "example.properties" file. 
Any concerns of having the about.jsp (file above) merged and updated?